### PR TITLE
Add retries to migrations and connections

### PR DESCRIPTION
**Pull request by c-du**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

File: central/centralhealth/service/service_impl.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/central/role"
+	versionUtils "github.com/stackrox/rox/central/version/utils"
```
Comment (dashrews78) [inline central/centralhealth/service/service_impl.go:12]: This is seems to be flagging changes that were in https://github.com/stackrox/stackrox/pull/3346/files.  Not sure if you need a rebase or what so we don't have to weed through the stuff you recently merged.

Comment (c-du) [reply central/centralhealth/service/service_impl.go]: Yes, rebased.

File: migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // This file was originally generated with
-// //go:generate cp ../../../../central/image/datastore/store/postgres/store.go store_impl.go
+// //go:generate cp ../../../../central/image/datastore/store/postgres/store.go store.go
```
Comment (dashrews78) [inline migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go:2]: Seems like most of these files are called postgres_plugin.  Why is this one different?

File: pkg/postgres/pgadmin/admin_utils.go [hunk 3]
```diff
@@ -256,14 +250,11 @@ func getPool(postgresConfig *pgxpool.Config) *pgxpool.Pool {
 	var err error
 	var postgresDB *pgxpool.Pool
 
-	if err := retry.WithRetry(func() error {
+	err = pgutils.Retry(func() error {
```
Comment (dashrews78) [inline pkg/postgres/pgadmin/admin_utils.go:253]: nice, that is a lot cleaner.

**Review by dashrews78 — APPROVED** [review]
Just the one nit question on file names in the migrations.

File: migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go (hunk 2 above)
Comment (c-du) [reply migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go:2]: postgres_plugin are auto generated and then unlinked. The store.go means the entire migration is customized.

**Pull request merged**